File: 1769. Minimum Number of Operations to Move All Balls to Each Box/brute-force/main.py
from typing import List


class Solution:
    def minOperations(self, boxes: str) -> List[int]:
       res = []

       for i in range(len(boxes)):
           total = 0
           for j in range(len(boxes)):
               if i == j: continue
               total += int(boxes[j]) * abs(i - j)
           res.append(total)

       return res

File: 1769. Minimum Number of Operations to Move All Balls to Each Box/brute-force/main.go
package main

import "math"

func minOperations(boxes string) []int {
	res := make([]int, len(boxes))

	for i := range boxes {
		total := 0
		for j := range boxes {
			if i == j {
				continue
			}
			if boxes[j] == '1' {
				total += int(math.Abs(float64(i - j)))
			}
		}
		res[i] = total
	}

	return res
}
